components/Card/Footer.d.ts

```diff
@@ -0,0 +1,9 @@
+import React = require('react');
```

Comment:
O que acha de importar assim:

```suggestion
import React from 'react';
```